README.md

```diff
@@ -352,23 +352,23 @@ import XpringKit
 let xrpPayIDClient = XRPPayIDClient(xrplNetwork: .main)
 
 let payID = 'georgewashington$xpring.money'
-xrpPayIDClient.xrpAddress(for: payID) { result in
-  switch result {
-  case .success(let xrpAddress):
-    print("Resolved to \(xrpAddress)")
-    print("")
-  case .failure(let error):
-    fatalError("Unknown error resolving address: \(error)")
-  }
+let result = xrpPayIDClient.xrpAddress(for: payID)
+switch result {
+case .success(let xrpAddress):
+  print("Resolved to \(xrpAddress)")
+case .failure(let error):
+  fatalError("Unknown error resolving address: \(error)")
 }
 ```
 
+Asynchronous APIs are also provided.
```

Comment:
This threw me off at first... should we provide demos for the asynchronous versions of these?

Reply:
We can always make a TODO to add these later if it's too much to get in by tomorrow.

Reply:
Filed https://app.asana.com/0/1143722888069470/1181926658900137 so that there's not a TODO hanging out in our docs.